Formula/tomcat.rb

```diff
@@ -22,6 +22,10 @@ def install
     (bin/"catalina").write_env_script "#{libexec}/bin/catalina.sh", JAVA_HOME: Formula["openjdk"].opt_prefix
   end
 
+  def post_install
+    ln_s etc/"tomcat", libexec/"config" unless (libexec/"config").exist?
+  end
+
```

Comment:
```suggestion

  def caveats
    <<~EOS
      Configuration files: #{pkgetc}
    EOS
  end

```
Let's do something like this for each formula too.